### DISCUSSION TITLE
Multiphoton Ionization

### DISCUSSION THREAD

**Discussion by gannettim** [SmileiPIC/Smilei#714] [Q&A]
Hi,
I was wondering if it is possible with Smilei to simulate multiphoton ionization through a laser concentrated in a focal plane where neutral gas is present. I saw that there is tunnel and collision ionization module, but not a multiphoton ionization module. 
Perhaps simulating a beam of photons passing through a gas could replicate such an ionization mode?

Thank you.

**Comment by mccoys** [comment]
There is the possibility to introduce a custom ionization rate, but in order to compute multiphoton ionization properly, one would need to develop a new module

**Reply by gannettim** [reply]
Hi mccoys, thanks for the quick response. Would you have any suggestions on its implementation?

**Reply by mccoys** [reply]
You first need a model for multiphoton ionization. I am not sure it is easy to find for a pic code. If the model depends only on the atom and the value of the EM field, I guess copying tunnel ionization and changing the ionization rate would work
